include/aws/common/byte_buf.h

```diff
@@ -440,6 +440,11 @@ const uint8_t *aws_lookup_table_to_lower_get(void);
 
 AWS_EXTERN_C_END
 
+/**
+ */
+#define AWS_BYTE_CUR_INIT_FROM_LITERAL(literal)                                                                  \
+    { .ptr = (uint8_t *)(literal), .len = AWS_ARRAY_SIZE(literal) }
```

Comment:
won't AWS_ARRAY_SIZE of "a" be 2? because of the '\0'?

Reply:
Yes! That's probably not what we want here huh.